### ISSUE TITLE
mouse build fails for v0.99.6a

### ISSUE THREAD

**Issue by ndaniel** [ndaniel/fusioncatcher#18]
`fusioncatcher-build -g mus_musculus -o fusioncatcher/data/mus_musculus`

The error log showed this:
```
--------------------------------------------------------------------------------------------
ERROR: Workflow execution failed at step 50 while executing:
----------------
   shield_genes.py \
   --organism mus_musculus \
   --read-len 500 \
   --output fusioncatcher/data/mus_musculus/ \
   --pseudo-genes-check
----------------


Executing second time the same step/command in order to capture error
messages (i.e. STDERR)...

-------------------------------------------
Traceback (most recent call last):
  File "fusioncatcher/bin/shield_genes.py", line 370, in <module>
    gis = set(gene_ids)
NameError: name 'gene_ids' is not defined
```

**Comment by ndaniel** [comment]
Manual fix
-------------

Change line no 195 of file `fusioncatcher/bin/shield_genes.py` from:

```
    if options.organism.lower() == 'homo_sapiens':
```

to

```
    gene_ids = []

    if options.organism.lower() == 'homo_sapiens':
```

**Comment by ndaniel** [comment]
Fixed and backported to v0.99.6a.
